lib/entrypoint-browser.ts

```diff
@@ -127,6 +127,28 @@ function defineAfterStep(
   }
 }
 
+function defineBeforeAll(fn: IHookBody): void;
+function defineBeforeAll(
+  maybeFn?: IHookBody
```

Comment:
The optionality and function overloading here is unecessary, as there's only one way to invoke this method.

Reply:
Makes sense, I'll change this here and in other places too